suites/tentacle/rados/tier-3_rados_test_5+2_ecpools.yaml

```diff
@@ -0,0 +1,653 @@
+      desc: Verify PG splitting and merging with network delay
+      polarion-id: CEPH-83571705
+      config:
+        create_pools:
+          - create_pool:
+              profile_name: ec52_9
+              pool_name: ec52_pool9
+              pg_num: 32
+              rados_put: true
+              num_objs: 200
+              byte_size: 1024
+              pool_type: erasure
+              k: 5
+              m: 2
+              crush-failure-domain: host
+              plugin: isa
+        add_network_delay: true
+        delete_pools:
+          - ec52_pool9
+
+  - test:
+      name: Verify scrub logs
+      module: test_scrub_log.py
+      polarion-id: CEPH-83575403
+      config:
+        # After the implementation of BZ#2320860 set the verify_log to true
+        verify_log: false
+        pool_configs:
+            - type: isa
+              conf: sample-pool-8
+        pool_configs_path: "conf/tentacle/rados/test-confs/pool-configurations.yaml"
+      desc: Verify that scrub & deep-scrub logs are captured in OSD logs
+
+  - test:
+      name: Autoscaler test - pool pg_num_min
+      module: pool_tests.py
+      polarion-id: CEPH-83573425
+      config:
+        verify_pg_num_min:
+          configurations:
+            pool-1:
+              profile_name: ec52_11
+              pool_name: ec52_pool11
+              pool_type: erasure
+              k: 5
+              m: 2
+              plugin: isa
+              crush-failure-domain: host
+              pg_num_min: 16
+              rados_write_duration: 50
+              rados_read_duration: 10
+              delete_pool: true
+      desc: Specifying pool bounds on pool Pgs - Verify pg_num_min
+
+  - test:
+      name: client pg access
+      module: test_client_pg_access.py
+      polarion-id: CEPH-83571713
+      config:
+        verify_client_pg_access:
+          num_snapshots: 20
+          num_objects: 250
+          configurations:
+            pool-1:
+              profile_name: ec52_12
+              pool_name: ec52_pool12
+              pool_type: erasure
+              k: 5
+              m: 2
+              plugin: isa
+              crush-failure-domain: host
+              disable_pg_autoscale: true
+      desc: many clients clients accessing same PG with bluestore as backend
+
+  - test:
+      name: Migrate data bw pools.
+      module: test_data_migration_bw_pools.py
+      polarion-id: CEPH-83574768
+      config:
+        pool-1-type: replicated
+        pool-2-type: erasure
+        pool-1-conf: sample-pool-1
+        pool-2-conf: sample-pool-5
+        pool_configs_path: "conf/tentacle/rados/test-confs/pool-configurations.yaml"
+      desc: Migrating data between different pools. Scenario-2. RE -> EC
+
+  - test:
+      name: Migrate data bw pools.
+      module: test_data_migration_bw_pools.py
+      polarion-id: CEPH-83574768
+      config:
+        pool-1-type: erasure
+        pool-2-type: replicated
+        pool-1-conf: sample-pool-5
```

Comment:
sample-pool-5 is a 2+2 pool. update the config file with new configs and update the suite file